src/schema/objects/modalities.yaml

```
@@ -35,3 +35,9 @@ motion:
 nirs:
   display_name: Near-Infrared Spectroscopy
   description: Data acquired with NIRS.
+ephys:
+  display_name: Electrophysiology
```

This should be more specific, as there are many types of electrophysiology in BIDS. This would be like calling MRI imaging. I would suggest something more descriptive, like Microelectrode electrophysiology.

---

Good point, if we were to stick to the domain of the BEP it would be

```suggestion
  display_name: Animal Electrophysiology
```

and really that would "scale" even to other primates like us ;-)  But also it might be applied (I think) to some not really animate samples in petri-dishes, so not quite appropriate so I do like the suggested one more:

"Microelectrode" is interesting as it was present in PR title(s) but not ever mentioned in current  BEP

```suggestion
  display_name: Microelectrode Electrophysiology
```

WDYT @bids-standard/bep032 ?

---

well, I need to name a file, so let's proceed with "Microelectrode" and then change if needed.

---

@dorahermes @yarikoptic what other types of electrophysiology are in BIDS? I know there's electroencephalography, but not electrophysiology. If we are to be more specific perhaps “intracranial electrophysiology”?

I know the scope says “animal” right there in the BEP title, but just like with MRI, it can actually be done in both. I think it's best to keep the animal scope for the BEP as is, but leave the modality species-agnostic for easier future tacking-on of more stuff.

---

iEEG, EEG, MEG and general physiological recordings like ECG. Current discussions are surrounding EMG. @yarikoptic did I miss anything?

---

AFAIK EEG and even more so iEEG could be considered types of electrophysiology (hence such a close correspondence to iEEG in this BEP). iEEG is also intracranial.

---

But iEEG, EEG, MEG are all electroencephalography... We don't use the term electrophysiology for it, and we don't do it because it's not commonly done.

---

note: the odd `anat`, `func` as corresponding to MRI only is IMHO the historical inconsistency we might want to dissolve at some point.  Given the real proximity of e.g. `ieeg` and what we have in `ephys` now I think we might indeed at some point really converge into something common (`ephys` or not) to both...  Overall I am with @dorahermes and to me providing specialization for `Electrophysiology` is more "conceptual" at this point to place it a little aside from iEEG and EEG both of which do have contacts to measure electrical properties from some natural tissue (physio).

---

If this issue comes back I would suggest opening a separate issue to talk about it.

---

@Remi-Gau @yarikoptic @dorahermes ok, moved the discussion here, chime in if I missed something in OP → https://github.com/bids-standard/bids-specification/issues/1800

---

Thank you! I will reference in description of the PR and resolve this conversation!